packages/admin-stories/src/docs/components/Table/Basics.stories.mdx

```diff
@@ -0,0 +1,86 @@
+import { Meta, Canvas, Story, Source } from "@storybook/addon-docs";
+
+import dedent from "ts-dedent";
+
+<Meta title="Docs/Components/Table/Basics" />
+
+# Table
+
+> Tables display information in a way that's easy to scan, so that users can look for patterns and insights.
+>
+> \- [MUI Docs](https://mui.com/components/tables/)
+
+<Canvas>
+    <Story id="stories-components-table-basic-table--basic-table" />
+</Canvas>
+
+`Table` has three required parameters: `data`, `totalCount` and `columns`.
+
+`data` contains the data that should be displayed in the table. It must be an array of objects, and each object must have a unique `id`.
+
+`totalCount` specifies the total number of items.
+It's only relevant if pagination is used. Then, the totalCount should be provided by the API.
+If **no pagination** is used, `totalCount` can simply be set to `data.length`.
+
+`columns` is an array that defines the structure of the table.
+It specifies which columns exist and how to render them.
+More information can be found in the [column section](#table-column).
+
+## Table Column
+
+### name
+
+The only required property of a column is its `name`.
+Per default, the `name` is matched to a property of `data`, which is then rendered:
+
+<Canvas>
+    <Story id="stories-components-table-table-column-props--table-column-name-prop" />
+</Canvas>
+
+Matching also works for nested properties if the `name` is separated by dots:
+
+<Canvas>
+    <Story id="stories-components-table-table-column-props--table-column-nested-name-prop" />
+</Canvas>
+
+### render
+
+Passing a function to the `render` prop overwrites the default rendering behavior.
+The render function can be used for altering the styling of a column or for rendering columns that don't have a corresponding prop in `data`:
+
+<Canvas>
+    <Story id="stories-components-table-table-column-props--table-column-render-prop" />
+</Canvas>
+
+### header
+
+The `header` property defines the heading of a column.
+It can be a string or a JSX element.
+
+If no header is set, the heading of a column is empty.
+
+<Canvas>
+    <Story id="stories-components-table-table-column-props--table-column-header-prop" />
+</Canvas>
+
+### visible
+
+`visible` can be used to hide a column.
+
+<Canvas>
+    <Story id="stories-components-table-table-column-props--table-column-visible-prop" />
+</Canvas>
+
+### cellProps & headerProps
+
+Internally, the `Table` uses Material UI's `TableCell` to render table headers and cells.
+`cellProps` and `headerProps` can be used to pass props to these internal elements.
+You can find more information in the [Material UI Docs](https://v4.mui.com/api/table-cell/).
+
+### sortable
+
+See [Sorting page](/story/docs-components-table-sorting--page#sorting)
```

Comment:
Maybe you could write one sentence explaining the prop. For instance: "Used for sorting the table. See ... for more information.